ansible/roles/filebeat/templates/filebeat.reference.yml

```diff
@@ -0,0 +1,2071 @@
+#
```

Comment:
Is this file used for runtime configuration, or is it just provided for a reference (as the name seems to suggest)? I am not aware of Filebeat, so I'm trying to neutrally gauge how deeply I need to review this file (or not).

Reply:
This file just provides a reference. I made the file external just in case any edits has to be done to it.